jsx/Filter.js

```diff
@@ -87,11 +115,13 @@ class Filter extends Component {
           columns={this.props.columns}
         >
           {this.renderFilterFields()}
+          <br/>
```

Comment:
:(

Reply:
Ahah, this won't be necessary once I rebase to #4203